sirepo/template/radia_util.py

```diff
@@ -304,6 +304,7 @@ def build_stl(**kwargs):
     g_id = radia.ObjPolyhdr(
         d.vertices, (numpy.array(d.faces) + 1).tolist(), d.magnetization
     )
+    g_id = radia.TrfOrnt(g_id, radia.TrfTrsl([d.center[0], d.center[1], d.center[2]]))
```

Comment:
This doesn't quite work on its own. We want to use the center defined by the user, irrespective of the points in the stl file. I think the easiest way is to subtract off the stl's center first:
```python
o.stlCentroid = mesh.centroid.tolist()
...
g_id = radia_util.build_stl(
...
    center=[x - o.stlCentroid[i] for i, x in enumerate(o.center)],
)
```

Reply:
Ah, I was testing radia.TrfOrnt() without realizing I was using (0,0,0) as my initial center, misunderstood the function.